railties/lib/rails/application.rb

```diff
@@ -400,6 +401,13 @@ def secrets
       end
     end
 
+    def credentials
+      @credentials ||= ActiveSupport::EncryptedConfiguration.new \
+        config_path: Rails.root.join("config/credentials.yml.enc"), 
```

Comment:
@dhh This is great that `ActiveSupport::EncryptedConfiguration` takes this file path as an argument.
Do you think we can expand it a little and make it configurable so one can easily work with few encrypted files, ie for "production (and derivative environments, like exposed betas)"?

And have some nice tooling like:
`bin/rails credentials:edit config/credentials-with-my-custom-name.yml.enc` that supports out of the box finding key with relative name?

I tried to bring it to secrets in https://github.com/rails/rails/pull/29909 but I can work on it here if you like the idea